src/scripts/settings.js

```diff
@@ -0,0 +1,167 @@
+const selectors = {
+    stateJson: '#state',
+    emptyState: '.empty-state',
+    deleteFormula: 'trash',
+    createFormula: 'create_formula',
+    pen: 'pen',
+    textInputforCreatingFormula: '.text_input_lol',
+    discard: '.discard',
+    save: '.save',
+    formulasFolder: '#formulasFolder',
+    template: '.formula-template',
+    createFormulaFromEmptyState: 'b_0',
+    link: 'linkToFormulasPage',
+    table: '.formulas_s',
+    home: '#home'
+}
+
+
+function getState() {
+    const data = localStorage.getItem('state');
+    return JSON.parse(data);
+}
+function updateState(state) {
+    localStorage.setItem('state', JSON.stringify(state));
+}
+
+const state = getState();
+if (state){
+    console.log("🔥#GOOD#🔥 ---> state.json parsed from localeStorage!!!")
+}
+
+
+
+
+
+
+
+//  CURRENT LIST OF FORMULAS
+const currentSettings  = document.querySelector(selectors.formulasFolder);
```

Comment:
I think we should be able to do it dynamically. We need to take a look at it later